backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -453,6 +453,158 @@ describe("Listings", () => {
     })
   })
 
+  // REMOVE_WHEN_EXTERNAL_NOT_NEEDED
+  describe("/listings/combined", () => {
+    it("should return all listings", async () => {
+      const res = await supertest(app.getHttpServer()).get("/listings/combined").expect(200)
+      expect(res.body.items.map((listing) => listing.id).length).toBeGreaterThan(0)
```

Comment:
I really want to avoid any situations where `backend/core` is creating external listings as much as possible because that should never happen.  I am going to add something to the seed script to generate some as a one-off and will modify the e2e tests to verify that `/listings/combined` returns more than `/listings`

Reply:
The test to compare results was added in the latest commit